lib/peep.rb

```diff
@@ -0,0 +1,24 @@
+require 'pg'
+
+class Peep
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
```

Comment:
There could be a method for this, in order to avoid repeating code!

Reply:
Good observation! If you hadn't commented on it yourself, I would have 😁